base_sequence_option/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
```

Comment:
```suggestion
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).

```

All of license should agpl?

Reply:
@Saran440 according to https://www.odoo-community.org/resources/faq AGPLv3, LGPLv3 or GPLv3 are possible.

Reply:
okay, i will change it next PR.
cc: @kittiu